--- sage-7.3/build/pkgs/python2/spkg-install	2016-08-28 17:15:28.292765003 -0700
+++ sage-7.3/build/pkgs/python2/spkg-install.new	2016-08-28 20:23:44.064494395 -0700
@@ -72,6 +72,8 @@
 
 build()
 {
+    LDFLAGS+=" -lcrypt -lreadline -lncurses "
+    export LDFLAGS
     ./configure --prefix="$SAGE_LOCAL" --libdir="$SAGE_LOCAL/lib" \
         --enable-unicode=ucs4 --enable-shared $PYTHON_CONFIGURE
 
